python/ffi_navigator/util.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from pathlib import Path
+
+
+def is_win():
+    return sys.platform == "win32"
+
+
+def join_path(root_path, relative_path):
+    return str(root_path / Path(relative_path))
+
+
+def normalize_path(raw_path):

```

Comment:
`normalize_path` looks a bit weird. Something like `unify_path` might be better?

Reply:
https://docs.python.org/3/library/os.path.html#os.path.normpath

Reply:
Gotcha. Then I'm fine with both `normalize_path`.